dictionaries/CallMap.php

```diff
@@ -13785,7 +13785,7 @@
 'stream_get_contents' => ['string|false', 'stream'=>'resource', 'length='=>'int', 'offset='=>'int'],
 'stream_get_filters' => ['array'],
 'stream_get_line' => ['string|false', 'stream'=>'resource', 'length'=>'int', 'ending='=>'string'],
-'stream_get_meta_data' => ['array{timed_out:bool,blocked:bool,eof:bool,unread_bytes:int,stream_type:string,wrapper_type:string,wrapper_data:mixed,mode:string,seekable:bool,uri:string,mediatype:string}', 'stream'=>'resource'],
+'stream_get_meta_data' => ['array{timed_out:bool,blocked:bool,eof:bool,unread_bytes:int,stream_type:string,wrapper_type:string,wrapper_data:mixed,mode:string,seekable:bool,uri:string,mediatype:string,crypto?:mixed}', 'stream'=>'resource'],
```

Comment:
Does psalm support the same logic internally for object-like arrays? Just want to be certain I can indicate this is a contextual return item `crypto?:mixed`.

Reply:
The same as what? If you're asking if you can have nested array shapes, then yes, you can.

Like this:
```
... seekable: bool, uri:string, mediatype:string, crypto?: array{protocol:string, cipher_name: string ... and so on}...
```